e2e/test.js

```diff
@@ -0,0 +1,51 @@
+const expect = require('chai').expect;
+
+describe('InstantSearch e-commerce demo', () => {
+  before(() => {
+    browser.maximizeWindow();
+  });
+
+  it('must work', async () => {
+    await browser.url(
+      'https://deploy-preview-3967--instantsearchjs.netlify.com/examples/e-commerce/'
+    );
+
+    // Select "Apple" brand in list
+    const brand = await browser.$('span=Apple');
```

Comment:
Can we rely on something than `span`? This seems flaky and we shouldn't describe where to get the information, but rather what information to get. In that case we want the `label` (this is what drives accessibility) element having as value `"Apple"`.

Reply:
The `label` element have the value `Apple\n442` and the linebreak seems to trigger some issues with the selector (I did not take the time to try to fix it).

But we can also use CSS class selectors if needed. https://webdriver.io/docs/selectors.html

Reply:
I wish to be able to use [ByLabelText](https://testing-library.com/docs/dom-testing-library/api-queries#bylabeltext) here :(

If `span` sounds flaky, IMO, the class names are less flaky since we try to maintain all the same class names across the flavors(which is far from a good way to test in accessibility-wise, and which is testing implementation details).

Reply:
Since we use them for E2E tests we can imagine the usage for `[test-id]` attribute in the DOM.

Reply:
Yes, that's also a possibility.

Reply:
test- attributes could be nice by I don't see how we can do it without altering every the DOM of every widget.
We're pretty lucky our css classes are pretty stable across flavours, using them should be good enough

Reply:
Maybe this can help: https://github.com/testing-library/testcafe-testing-library.

Reply:
Sadly this is not compatible with WebdriverIO :disappointed: